ansible/files/postgresql_config/supautils.conf.j2

```diff
@@ -7,6 +7,6 @@ supautils.privileged_extensions = 'address_standardizer, address_standardizer_da
 supautils.privileged_extensions_custom_scripts_path = '/etc/postgresql-custom/extension-custom-scripts'
 supautils.privileged_extensions_superuser = 'supabase_admin'
 supautils.privileged_role = 'postgres'
-supautils.privileged_role_allowed_configs = 'auto_explain.log_min_duration, auto_explain.log_nested_statements, log_lock_waits, log_min_messages, pg_net.batch_size, pg_net.ttl, pgaudit.log, pgaudit.log_catalog, pgaudit.log_client, pgaudit.log_level, pgaudit.log_relation, pgaudit.log_rows, pgaudit.log_statement, pgaudit.log_statement_once, pgaudit.role, pgrst.*, plan_filter.*, safeupdate.enabled, session_replication_role, track_io_timing'
+supautils.privileged_role_allowed_configs = 'auto_explain.log_min_duration, auto_explain.log_nested_statements, log_lock_waits, log_min_messages, log_temp_files, pg_net.batch_size, pg_net.ttl, pgaudit.log, pgaudit.log_catalog, pgaudit.log_client, pgaudit.log_level, pgaudit.log_relation, pgaudit.log_rows, pgaudit.log_statement, pgaudit.log_statement_once, pgaudit.role, pgrst.*, plan_filter.*, safeupdate.enabled, session_replication_role, track_io_timing'
```

Comment:
We really need to push these to be handled via https://supabase.com/docs/guides/platform/custom-postgres-config , rather than supautils

If you update them using supautils, they won't get replicated to replicas (and in the future, failover primaries).

Reply:
Some of our settings `work_mem` and `max_parallel_workers` can be set through both the CLI and just generic SQL. For the time being, this is a fast way to make configs accessible. 

> " they won't get replicated to replicas (and in the future, failover primaries)."

Is this true only for `supautils` enabled settings or for any change outside of the CLI? Either or, a note should be added to the [customizing postgres docs](https://supabase.com/docs/guides/database/custom-postgres-config)

Reply:
I changed pg_audit's settings, which is handled by supautils. It captured data from both the primary and replica. Has there been any recent updates?